stl/inc/xstring

```diff
@@ -4225,19 +4234,19 @@ public:
         _Mypair._Myval2._Orphan_all();
     }
 
-private:
-    void _Swap_proxy_and_iterators(basic_string& _Right) {
-        _Mypair._Myval2._Swap_proxy_and_iterators(_Right._Mypair._Myval2);
-    }
-
     _Alty& _Getal() noexcept {
         return _Mypair._Get_first();
     }
 
-    const _Alty& _Getal() const noexcept {
+    const _Alty& _Getal() const noexcept { // used by operator+
         return _Mypair._Get_first();
     }
 
+private:
+    void _Swap_proxy_and_iterators(basic_string& _Right) {

```

Comment:
Observation, no change requested: reordering these member functions is OK because they're non-virtual.